Article/Article.js

```diff
@@ -5,28 +5,34 @@ class Article {
     // assign this.element to the passed in article element
     this.element = element;
     // create a reference to the ".expandButton" class. 
-    this.expandButton = document.querySelector('.expandButton');
+
+    
+    this.expandButton = document.getElementsByClassName('expandButton');
```

Comment:
I like this method that you are using. It makes it very clear that you are choosing a class.